README.md

```diff
@@ -1,2 +1,61 @@
-# leaderBoard
-The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.
+![](/To-do-list.png)
+
+# To Do List
+
+> This project is a part of microverse training program. It was build using HTML5, CSS, JavaScript and webpack.
+It is a web application that creates a To do list. In this project I implemented the following:
+
+1) Add webpack to the project
+2) Added an index.html that contains markup for the page
+3) Added an index.js file that contains array of object of the various to do activities.
+4)Added style.css file in src/ with styling for the page.
+5) Added and remove function to add and remove items from the array
+6) Added edit function to edit the various task
+7) Use webpack to bundle all the files in the src/ folder into dist/ folder that contains index.html and main.js.
```

Comment:
- [ ] Kindly, update your the description of your project and the project name 💯
> Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md). 👍